falco/values.yaml

```diff
@@ -15,6 +15,16 @@ containerd:
   enabled: true
   socket: /run/containerd/containerd.sock
 
+
+# if you cannot download or compile the driver provided by falco.
+# use this initContainer to build it with your own image.
+#
+# driverbuilder:
+#  initcontainer:
+#    - name: init-driverloader
+#      image: mydockerimage
+#      command: ["sh", "-c", "./buildmykit"]
+#      ... extra params to initContainer.
 resources:
```

Comment:
Although I like the idea to have an init container, I think this doc comment is too vague here (since we are not providing any init container for that).

Thus I'd propose to promote the `initcontainer` object to the top-level (removing the `driverbuilder` node, as suggested in the previous comments) in order to make it more general. At the same time, I'd add a more specific example that explains to the user how to leverage this feature to solve the use case you mentioned. 

WDYT?

Reply:
Also please see https://github.com/falcosecurity/charts/issues/36 (another way to this would as well as Grafana did in its chart :point_right: https://github.com/helm/charts/pull/12343 )

Reply:
Hi @leogr thanks for the review :)

sounds good to me as well. Let me work on that and i'll come back to you, i created the driverbuilder  to mark the intent clearer and because in some cases an initcontainer was added already.

Reply:
Great!

Yep, I just like the idea of "extraInitContainers" because it opens many possibilities, including the one you need :)

Reply:
@leogr i did your requested changes and tried it in my cluster and seems to work, can you take  a look again please?

Thanks!

Reply:
It looks great! Thank you!!! :clap: 

I just need some time to test it, I will do that ASAP